unidist/core/backends/mpi/core/communication.py

```diff
@@ -734,28 +686,7 @@ def isend_complex_operation(
         }
 
 
-def send_remote_task_operation(comm, operation_type, operation_data, dest_rank):
```

Comment:
Why is this method removed?

Reply:
This function has the same functionality as the send_complex_operation function. I replaced all usage and this function is no longer used.  I think this function should be removed to avoid duplication.